packages/ui/cart/drawer/Upsell.tsx

```diff
@@ -42,14 +44,19 @@ export function Upsell({ item }: UpsellProps) {
             borderColor="brand.200"
             borderStyle="solid"
          >
-            <Link href={item.url} color="brand.500" fontWeight="semibold">
-               {item.marketingTitle ?? item.name}
-            </Link>
-            <Text color="brand.500">Add the product below to your order</Text>
+            <NextLink href={`/products/${item.handle}`} passHref>
+               <LinkOverlay
+                  id={item.handle}
```

Comment:
I'm curious why we're including an id here?

Reply:
i copied pasted this and i just assumed that the id was a good thing to use.

Reply:
Gotcha. I don't think it's wrong, but I also don't think it does anything here and we don't include id's on most components.